HW0/P5/P5.txt

```diff
@@ -1 +1,17 @@
 # Your answers here.
+
+# The speed is: 1 [seconds/(add*person)]
+
+# 1. For a single cashier: 255 additions in series = 255 seconds.
+
+# 2. For 7 cashiers: They can split the 256 bags up in 3 piles of 36 and 4 piles of 37. This will take 36 seconds. Then they have 7 numbers, which in 1 second can be reduced to 4 by 3 of them, to 2 in another second by 2 of them, and to the result by one of them in one second. The total is then 36+3 = 39 seconds. 
+
+# 3. For infinite cashiers: 256 = 2**8, so 8 sequential additions of two numbers are necessary and so it takes 8 seconds. (First requiring 128, then 64, then 32,...then one employee)
```

Comment:
I really like the concise explanation you give for your responses throughout P5 but this one is my favorite. **Good job**